etc/profile-a-l/gtk-pipe-viewer.profile

```diff
@@ -0,0 +1,14 @@
+# Firejail profile for gtk-pipe-viewer
+# Description: Gtk front-end to pipe-viewer
+# This file is overwritten after every install/update
+# Persistent local customizations
+include gtk-pipe-viewer.local
+# added by included profile
+#include globals.local
+
+ignore quiet
+
+include whitelist-runuser-common.inc
```

Comment:
Why only for gtk-pipe-viewer?

Reply:
gtk-straw-viewer and gtk-youtube-viewer already exist like this.

Reply:
And what about pipe-viewer, straw-viewer and youtube-viewer?

Reply:
xxxx-viewer are terminal applications, the original author of the profiles probably put quiet, so that you don't get assaulted by a wall of text, every time you launch it. gtk-xxxx-viewer are gui applications, so it doesn't matter if the terminal gets verbose. 

I think it's best to leave it like this

Reply:
I mean wruc, GH always adds the three line above on one-line comments.

Reply:
i don't know why that is there, i don't see anything broken with or with out it, but i'm not using all features either. So i'm been conformist, so that nothing brakes.

Reply:
rm wruc